projects/aws/bottlerocket-bootstrap/pkg/kubeadm/controlplane_init.go

```diff
@@ -124,9 +126,29 @@ func controlPlaneInit() error {
 		return errors.Wrap(err, "Error waiting for kubelet to come up")
 	}
 
+	cmd = exec.Command(kubeadmBinary, utils.LogVerbosity, "version", "-o=short")
+	out, err = cmd.CombinedOutput()
+	if err != nil {
+		return errors.Wrapf(err, "Error running command: %v, Output: %s\n", cmd, string(out))
+	}
+	kubeadmVersion, err := versionutil.ParseSemantic(string(out))
+	if err != nil {
+		return errors.Wrapf(err, "%q is not a valid kubeadm version", string(out))
+	}
+
+	compare, err := kubeadmVersion.Compare("1.26")
```

Comment:
We probably want this to be >= 1.26.

In general, this is probably fine to get the version from the kubeadm binary, but i suspect the kube version would also be in the kubeadm config. Not sure if its necessarily a better spot to get it from, but maybe we are already getting this somewhere else in the code? thoughts @vignesh-goutham

Reply:
Yes, if it's less than 1.26, we don't skip the new phase (bc it didn't exist in the older versions), but if 1.26+, we skip the new phase.

But yes, open to other thoughts if we dont want to use `kubeadm version`